hummingbot/strategy/pure_market_making/constant_size_sizing_delegate.pyx

```diff
@@ -37,17 +49,30 @@ cdef class ConstantSizeSizingDelegate(OrderSizingDelegate):
             bid_order_size = market.c_quantize_order_amount(market_info.symbol, self.order_size)
             ask_order_size = market.c_quantize_order_amount(market_info.symbol, self.order_size)
 
+        if (bid_order_size ==0):
+            self.logger().network("", f"Buy(bid) order size is less than minimum order size. Buy order will not be placed")
```

Comment:
If you're doing the recoverable error log, the first message should always be non-empty - since that goes to the logs and should be useful for debugging.

The second message (user warning) you have right now looks like it should be the first message - since it pinpoints the reason for the error. The user warning should be something that the user can act on (e.g. the order size config is too small for the market). So instead of something unhelpful like "order will not be placed", it should be a suggestion to something the user can do (e.g. check the order size configs.)